server/src/test/java/org/opensearch/identity/IdentityPluginTests.java

```diff
@@ -6,12 +6,11 @@
  * compatible open source license.
  */
 
-package org.opensearch.bootstrap;
+package org.opensearch.identity;
```

Comment:
Integration tests could be illustrative of what scopes are allowing/disallowing, while I'm not sure of all the technical components I'd like to see tests cases with a structure like the following:

```java 
public void testReadScopeEnforced() {
   // Setup
   testApp.setApplicationScopes(List.of());
   cluster.register(testApp);
   // Action
   final UnauthorizedException ue = assertException(UnauthorizedException.class, () -> testApp.readFromIndex(cluster))
   // Verification
   assertThat(ue.getPermission(), containsInAnyOrder("Action.Index.Read", "Action.Index.All");
   verify(testApp).readFromIndex();
   verifyNoMoreInteractions(testApp);
}
```

Reply:
Yeah, I tried to set this up in the ShiroSubjectTests above. 

I can add some more here using a different application view.